go.mod

```diff
@@ -1,33 +1,33 @@
 module github.com/openshift/machine-api-provider-nutanix
 
-go 1.21
+go 1.22.0
 
-toolchain go1.21.3
+toolchain go1.22.4
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/coreos/ignition/v2 v2.14.0
 	github.com/go-logr/logr v1.4.1
 	github.com/golang/mock v1.6.0
 	github.com/nutanix-cloud-native/prism-go-client v0.2.0
-	github.com/onsi/ginkgo/v2 v2.15.0
-	github.com/onsi/gomega v1.31.1
-	github.com/openshift/api v0.0.0-20240124164020-e2ce40831f2e
-	github.com/openshift/machine-api-operator v0.2.1-0.20240125175440-c9de8bda0dd1
+	github.com/onsi/ginkgo/v2 v2.17.1
+	github.com/onsi/gomega v1.33.0
+	github.com/openshift/api v0.0.0-20240530053948-b01900f1982a
```

Comment:
This update includes moving CRDs into a `zz_generated.crd-manifests` folder below were they previously were. You'll need to use the `tools.go` to import this folder and then update all envtest imports for the CRDs to point to the new location

Reply:
I did this and now the tests are failing because the API server can't find the `isIP` CEL function.

That appears to have been introduced at https://github.com/openshift/api/pull/1873/files#diff-07b264a49084976b670fb699badaca1795027d6ea732a99226a5388104f6174fR894, and per https://github.com/kubernetes/kubernetes/pull/121912, it shouldn't be supported until Kube 1.31.

I find it hard to believe that I'm the first to hit this issue, though, so going to do some more digging to see if anyone else has and what the solution was.

Reply:
Found https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/356/commits/89c7d105e58cbe4e13274e865c8bb8a62a7463ae, testing out similar changes.